backstop/tests/color-picker.html

```diff
@@ -0,0 +1,190 @@
+<!--
+  Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+  See LICENSE.md in the project root for license terms and full copyright notice.
+-->
+<!DOCTYPE html>
+<html
+  lang="en-US"
+  id="theme"
+>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1"
+    >
+    <title>color-picker | iTwinUI</title>
+    <link
+      rel="stylesheet"
+      href="../../lib/css/all.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../assets/demo.css"
+    />
+    <script src="../assets/theme.js"></script>
+  </head>
+  <body class="iui-body">
+    <theme-button></theme-button>
+    <h1>color-picker</h1>
+    <hr />
+
+    <h2>Basic</h2>
+    <section id="demo-default">
+      <div class="iui-color-picker">
+        <div
+          style="background: #FFF;"
+          class="iui-color-option iui-light"
+          tabindex="0"
+        ></div>
+        <div
+          style="background: #00121D;"
```

Comment:
You could use an inline [CSS variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) instead of setting inline background.

See example in [`badge.html`](https://github.com/iTwin/iTwinUI/blob/main/backstop/tests/badge.html)